applications/serial_lte_modem/overlay-carrier.conf

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_LWM2M_CARRIER=y
```

Comment:
I think this confuguration should be added to the `sample.yaml` file so that it does not get bitrotten over time.